From 10a44e6468baa7095ccd3d28e27aa15eaadce9e7 Mon Sep 17 00:00:00 2001
From: Frederic Crozat <fcrozat@mandriva.com>
Date: Mon, 28 Sep 2009 14:14:46 +0200
Subject: [PATCH] fix missing parameter for creat

---
 rs232int.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/rs232int.cpp b/rs232int.cpp
index 7951cb9..fc3e842 100644
--- a/rs232int.cpp
+++ b/rs232int.cpp
@@ -143,7 +143,7 @@ int RS232Interface::OpenSerial(int no)
 		snprintf(lockname, MAXPATH, "%s/LCK..%s%d", profile->GetLockDir(), profile->GetDevName(), no);
 		UserDebug1(UserApp2, "RS232Interface::OpenSerial() now lock the device %s\n", lockname);
 
-		fd = open ((const char *)lockname,O_RDWR|O_EXCL|O_CREAT);
+		fd = open ((const char *)lockname,O_RDWR|O_EXCL|O_CREAT, S_IRWXU);
 		if (fd < 0)
 		{
 			fd = open ((const char *)lockname,O_RDONLY);
-- 
1.6.4.4

